riscv/dts.cc

```diff
@@ -125,7 +125,7 @@ std::string dts_compile(const std::string& dts)
     close(dts_pipe[1]);
     close(dtb_pipe[0]);
     close(dtb_pipe[1]);
-    execl(DTC, DTC, "-O", "dtb", 0);
+    execl(DTC, DTC, "-O", "dtb", NULL);
```

Comment:
Needs a `(char *)` cast in front of it since this is a variadic function and `NULL` may not expand to something of pointer type (just something that is a null pointer constant in a pointer context).

Reply:
Fixed.  FreeBSD has a note further down in BUGS about needing the char * for portability.  FreeBSD does use '(void *)0' as NULL except when compilers have a built-in null.

Reply:
Ah I see, though that's not quite correct. `(void *)0` is actually *not* a null pointer constant in *C++* and thus if you're using an old compiler, `NULL` is often just `0` (or `0L`); see `sys/_null.h` on FreeBSD. So yes, if you're writing C then `NULL` is *probably* ok (though it technically could still be defined to just `0`), but it gets more dangerous in C++.